### PR TITLE
Julia/delete repository final

### PR DESCRIPTION
**Pull request by julialieberman**
Closed the other branches that did this. This is now branched off of dev and works both in input bar and on right click! issue #13

### CODE REVIEW

File: commands/azureCommands/delete-repository.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    let answer = await vscode.window.showInputBox(opt);
+    answer = answer.toLowerCase();
+    if (answer !== 'yes') { return; }
+
+    // generate credentials before requesting a delete.
+    if (context) {
+        username = context.userName;
+        password = context.password;
+        repoName = context.label;
+        subscription = context.subscription;
```
Comment (northtyphoon) [inline commands/azureCommands/delete-repository.ts]: The subscription is not used. You can remove it.

File: commands/azureCommands/delete-repository.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    let username: string;
+    let password: string;
+    if (!context) {
+        registry = await quickPicks.quickPickACRRegistry();
+        subscription = acrTools.getRegistrySubscription(registry);
+        let repository: Repository = await quickPicks.quickPickACRRepository(registry);
+        repoName = repository.name;
+    }
+
+    //ensure user truly wants to delete registry
```
Comment (northtyphoon) [inline commands/azureCommands/delete-repository.ts]: nit: format: add a space between `//` and `ensure`

File: commands/azureCommands/delete-repository.ts (hunk 1 above)
Comment (rsamai) [reply commands/azureCommands/delete-repository.ts]: It's actually used in the following else statement
 let creds = await acrTools.loginCredentials(subscription, registry);

**Comment by estebanreyl** [comment]
I think this branch is ready now. Although the same weird thing happened where it thinks it has more changes than it actually has?

**Pull request closed without merging**